src/scss/imports/_angular-material-override.scss

```diff
@@ -0,0 +1,252 @@
+a.md-button,
```

Comment:
I'm not sure we need this override, it looks more like killing angular material rather then override.